src/site/xdoc/key_duration.xml

```diff
@@ -30,6 +31,19 @@ In datetime maths you could say:
 </p>
 <p>
 Durations implement <code>Comparable</code> which compares the lengths of the two durations.
+Be advised that comparing durations with opposite order, yields unexpected results:
```

Comment:
The result is never unexpected.  It's postive, negative, or zero for a defined set of rules.

Reply:
I've updated wording, please check now.